optuna/samplers/_tpe/multivariate_parzen_estimator.py

```diff
@@ -0,0 +1,371 @@
+from typing import Dict
+from typing import Optional
+from typing import Tuple
+
+import numpy as np
+import scipy.special
+from scipy.stats import truncnorm
+
+from optuna import distributions
+from optuna.distributions import BaseDistribution
+from optuna.samplers._tpe.parzen_estimator import _ParzenEstimatorParameters
+
+EPS = 1e-12
+_DISTRIBUTION_CLASSES = (
+    distributions.UniformDistribution,
+    distributions.LogUniformDistribution,
+    distributions.DiscreteUniformDistribution,
+    distributions.IntUniformDistribution,
+    distributions.IntLogUniformDistribution,
+    distributions.CategoricalDistribution,
+)
```

Comment:
We already have the `DISTRIBUTION_CLASSES` in `optuna/distributions.py`.

Reply:
I believe the partially shared consensus was to maintain a list of supported distributions by multivariate TPE separately, confined to multivariate TPE, without being affected by changes to `distributions.py`.

Reply:
I chose to define this tuple at the module level, in case modifications in `optuna/distributions.py` are made in the future.
What do you think about this point?

Reply:
Thanks for the clarification. I understood. It's fine as it is now.